src/coreclr/src/inc/holder.h

```diff
@@ -608,7 +608,7 @@ class BaseWrapper : public BaseHolder<TYPE, BASE, DEFAULTVALUE, IS_NULL>
     {
         return !!(this->m_value != TYPE(value));
     }
-#ifdef __GNUC__
+
     // This handles the NULL value that is an int and clang
```

Comment:
A nit - the comment talks about clang, but it should be changed to "the compiler"